styles.css

```diff
@@ -375,18 +394,26 @@ hr {
   .middle-section {
     align-items: flex-start;
     flex-direction: column;
+    grid-template-columns: 650px;
+    grid-template-rows: 790px;
+    box-sizing: border-box;
   }
 
   .card1 {
     display: block;
-    width: 90%;
     background-size: contain;
-    height: 80%;
-    margin-right: 80px;
+    background-color: #fff;
+    border: 1px white;
+    border-radius: 16px;
+    height: 700px;
+    width: 550px;
+    align-self: flex-start;
+    margin-top: 88px;
+    padding-right: 5px;
   }
 
   .responsive {
-    width: 360px;
+    width: 450px;
```

Comment:
Please your project is not responsive to different screen size 
![respon](https://user-images.githubusercontent.com/55596970/169627962-bc6b207c-1b51-46a4-a056-d78da0f4f94e.png)